hooks/pre/12-clear_pulp_content.rb

```diff
@@ -0,0 +1,73 @@
+require 'fileutils'
+
+def clear_pulp_content
+  content_dir = '/var/lib/pulp/content'
+  if File.directory?(content_dir)
+    logger.debug "Removing Pulp content inside \'#{content_dir}\'."
+    FileUtils.rm_rf('/var/lib/pulp/content/*')
```

Comment:
https://ruby-doc.org/stdlib-2.4.1/libdoc/fileutils/rdoc/FileUtils.html#method-c-rm_r suggests you need to call glob explicitly.
```suggestion
    FileUtils.rm_rf(Dir.glob(File.join(content_dir, '*')))
```

I'm also wondering about the docs for it that mention a security risk. I guess this is a timing attack where it would start to process it, read a directory and then replace that with a symlink. I wouldn't be too worried about this.